modules/images-configuration-parameters.adoc

```diff
@@ -47,6 +47,28 @@ pods. For instance, whether or not to allow insecure access. It does not contain
 
 Either `blockedRegistries` or `allowedRegistries` can be set, but not both.
 
+|`imageStreamImportMode`
+|Controls the import mode behaviour of image streams. You can set it to either of the following values:
+
+* `Legacy`: Indicates that the legacy behaviour must be used. The legacy behaviour discards the manifest list and imports a single sub-manifest. In this case, the platform is chosen in the following order of priority: 
+. Tag annotations: Determining the platform by using the platform-specific annotations in the image tags.
+. Control plane architecture or the operating system: Selecting the platform based on the architecture or the operating system of the control plane.
+. `linux/amd64`: If no platform is selected by the above methods, the `linux/amd64` platform is selected. 
+. The first manifest in the list is selected.
+
+// What uses the behavior? What determines/Selects?
```

Comment:
When a tag is created in an imagestream, the importmode can be specified as part of the importpolicy : https://github.com/openshift/api/blob/b1a9b9a2e7feee7d6e9fcbc2d13641cd399dbd75/image/v1/types.go#L303 which controls how the image is imported for that tag (single manifest vs manifestlist). The default used to be Legacy. This feature changes that so that the importmode is configured implicitly based on the architecture field in the ClusterVersion's desired status. "multi" would indicate the use of PreserveOriginal as import mode which would import the manifestlist and no entry or an empty architecture field would indicate Legacy which is to import the single manifest : https://github.com/openshift/api/blob/b1a9b9a2e7feee7d6e9fcbc2d13641cd399dbd75/config/v1/types_cluster_version.go#L741